core/src/spotify_id.rs

```diff
@@ -24,6 +24,21 @@ pub enum SpotifyItemType {
     Unknown,
 }
 
+impl ToString for SpotifyItemType {
```

Comment:
This somehow seems redundant when we already have `From<&str>`? The place where you're calling `to_string` you could probably do the conversion differently.

Reply:
No problem I can get rid of it.

Reply:
The only way I can convert it to a string is with `format!("{:?}", item_type)` which works I guess for the event handler (that's the only thing that needs a string).

Reply:
Alrighty then.